pwndbg/commands/search.py

```diff
@@ -254,8 +243,7 @@ def search(
         value += b"\x00"
 
     elif type == "asm" or asmbp:
-        bits_for_arch = pwnlib.context.context.architectures.get(arch, {}).get("bits")
-        value = pwnlib.asm.asm(value, arch=arch, bits=bits_for_arch)
+        value = pwndbg.aglib.asm.asm(value)
```

Comment:
With the previous code we allowed for search cross-arch instructions... but idk if that feature was used by anyone

Reply:
🤷